flags.go

```diff
@@ -373,17 +379,18 @@ type flagStorage struct {
 	SequentialReadSizeMb               int32
 
 	// Tuning
-	MaxRetrySleep       time.Duration
-	StatCacheCapacity   int
-	StatCacheTTL        time.Duration
-	TypeCacheTTL        time.Duration
-	MaxRetryDuration    time.Duration
-	RetryMultiplier     float64
-	LocalFileCache      bool
-	TempDir             string
-	DisableHTTP2        bool
-	MaxConnsPerHost     int
-	MaxIdleConnsPerHost int
+	MaxRetrySleep         time.Duration
+	StatCacheCapacity     int
+	StatCacheTTL          time.Duration
+	TypeCacheTTL          time.Duration
+	MaxRetryDuration      time.Duration
+	RetryMultiplier       float64
+	LocalFileCache        bool
+	TempDir               string
+	DisableHTTP2          bool
+	MaxConnsPerHost       int
+	MaxIdleConnsPerHost   int
+	EnableNonexistentType bool
```

Comment:
Can we rename EnableNonexistentType to EnableNonexistentTypeCache here and everywhere?

Reply:
will do. Thanks for the feedback.

Reply:
Done!